lib/src/current_weather/data/adapters/current_weather_from_dto.dart

```diff
@@ -12,7 +12,7 @@ extension CurrentWeatherFromDto on CurrentWeatherDto {
       updatedAt: current.lastUpdated,
       image: imageUrl,
       wind: current.windKph,
-      perceivedtemp: current.feelslikeC,
+      perceivedTemp: current.feelslikeC,
```

Comment:
Just for fun: look at these two identical lines. Git is weird sometimes 😆

Reply:
There's a capital T in perceivedTemp :)

Reply:
woops 😄